model/message.go

```diff
@@ -11,7 +11,7 @@ type Message struct {
 	Message       string `gorm:"type:text"`
 	Title         string `gorm:"type:text"`
 	Priority      int
-	Extras        []byte
+	Extras        []byte `gorm:"type:blob"`
```

Comment:
Seems like postgres doesn't like `blob`:
```
(pq: type "blob" does not exist) 
[2019-06-08 08:59:57]  
[GIN-debug] [WARNING] Running in "debug" mode. Switch to "release" mode in production.
 - using env:	export GIN_MODE=release
 - using code:	gin.SetMode(gin.ReleaseMode)

panic: pq: relation "plugin_confs" does not exist

goroutine 1 [running]:
github.com/gotify/server/router.Create(0xc0000b2240, 0xc00009e240, 0xc0000ee000, 0x52, 0xc0000b05a0)
	/proj/router/router.go:46 +0x311f
main.main()
	/proj/app.go:51 +0x382
```

Reply:
Can you try varbinary and see if that works?

Reply:
varbinary gives me the same error. `bytea` works for postgres but not for mysql.

Maybe we need some custom alter table if dialect == "mysql" but that would only be a workaround.

Reply:
wo could also convert the byte array to a string and then save it as text which is supported in both databases.

Reply:
So we should not specify the column type in the struct tag but to use an ALTER manually, right?

Reply:
I think we could get into encoding issues by storing binary data as text :(

Reply:
> So we should not specify the column type in the struct tag but to use an ALTER manually, right?

Yup

> I think we could get into encoding issues by storing binary data as text :(

The binary data could be converted to base64 or hex before storing it in the database.

Reply:
Can you try if this one works? I tried sqlite3 and mariaDB